Modules/xASL_module_BIDS2Legacy.m

```diff
@@ -56,8 +56,17 @@
         diary(x.dir.diaryFile); % Start diary
     end
     
-    x = xASL_init_InitializeMutex(x, 'BIDS2Legacy'); % Start Mutex
+    [x, bLocked] = xASL_init_InitializeMutex(x, 'BIDS2Legacy'); % Start Mutex
```

Comment:
Same here - we shouldn't use bLocked parameter.

Reply:
Now throughout bLocked -> x.mutex.bAnyModuleLocked

Reply:
Sounds good. Will check and test in detail.